ads/adocean.js

```diff
@@ -61,9 +61,7 @@ function setupAdoConfig(mode, global, consent) {
 function setupPreview(global, data) {
   if (global.ado && data.aoPreview && !isFalseString(data.aoPreview)) {
     global.ado.preview({
-      enabled: true,
-      emiter: data['aoEmitter'],
-      id: data['aoPreview'],
+      enabled: true
```

Comment:
Looks like a small lint error here. You can try `gulp lint --fix` or if you use vscode we have some docs [here](https://github.com/ampproject/amphtml/blob/cea95ff3c5436ac0f0ca620fff269634cdeb535a/contributing/getting-started-e2e.md#workflow-for-visual-studio-code) to integrate the linting into the editor. Many other editors also have similar functionality.